.github/workflows/linux.yml

```diff
@@ -208,6 +208,47 @@ jobs:
         cd tools
         pytest --pyargs numpy -m "not slow"
 
+  array_api_tests:
+    needs: [smoke_test]
+    runs-on: ubuntu-latest
+    if: github.event_name != 'push'
+    steps:
+    - name: Checkout NumPy
+      uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      with:
+        submodules: recursive
+        fetch-depth: 0
+    - name: Checkout array-api-tests
+      uses: actions/checkout@v3
+      with:
+        repository: data-apis/array-api-tests
+        ref: '83f0bcdcc5286250dbb26be5d37511702970b4dc'  # Latest commit as of 2023-11-15
+        submodules: 'true'
+        path: 'array-api-tests'
+    - name: Fix array-apis bug
+      # Temporary workaround for https://github.com/data-apis/array-api/issues/631 
```

Comment:
This should have been fixed yesterday, can you check if things pass now without this fixup with the latest commit from `array-api-tests`?

Reply:
Yes, it's fixed now.